WinUIGallery/ControlPages/Typography.xaml

```diff
@@ -0,0 +1,116 @@
+<!--
```

Comment:
Would it be better to move the Design guidance pages into a separate folder?

Reply:
Yes I agree, this would make more sense.